x-pack/qa/smoke-test-watcher-with-security/src/test/java/org/elasticsearch/smoketest/SmokeTestWatcherWithSecurityIT.java

```diff
@@ -238,8 +236,7 @@ public void testSearchTransformInsufficientPermissions() throws Exception {
 
         getWatchHistoryEntry(watchId);
 
-        Response response = adminClient().performRequest("GET", "my_test_index/doc/some-id",
-                Collections.singletonMap("ignore", "404"));
```

Comment:
isn't the ignore param needed?

Reply:
I believe that comes for free with HEAD.

Reply:
oh right!